src/plot.py

```diff
@@ -181,6 +184,23 @@ def show_same_smoothed(same, stop_at=1, header="smoothed", show_df=True):
                 break
     return average.T
 
+def smooth(study, runs, header=""):
+    loss, metric = study
+
+    avg_loss = sum([loss[i] for i in runs])/len(runs)
+    avg_metric = sum([metric[i] for i in runs])/len(runs)
+
+    avg_loss.plot(title="Loss "+header, figsize=(15,5))
+    avg_metric.plot(title="Metric "+header, figsize=(15,5))
+
+    print("Final values:")
+    display(avg_loss.tail(3))
+    display(avg_metric.tail(3))
+
+    return avg_loss, avg_metric
```

Comment:
For plotting multiple same runs (see issue #33)

Also displays the data frames (pretty print) of averages in notebook